vg/draw/canvas.go

```diff
@@ -549,30 +549,30 @@ func clipPoly(in func(Point, Point) bool, clip, norm Point, pts []Point) (clippe
 // slop is some slop for floating point equality
 const slop = 3e-8 // ≈ √1⁻¹⁵
 
-func isLeft(p, clip Point) bool {
+func isLeft(p, clip vg.Point) bool {
 	return p.X <= clip.X+slop
 }
 
-func isRight(p, clip Point) bool {
+func isRight(p, clip vg.Point) bool {
 	return p.X >= clip.X-slop
 }
 
-func isBelow(p, clip Point) bool {
+func isBelow(p, clip vg.Point) bool {
 	return p.Y <= clip.Y+slop
 }
 
-func isAbove(p, clip Point) bool {
+func isAbove(p, clip vg.Point) bool {
 	return p.Y >= clip.Y-slop
 }
 
 // isect returns the intersection of a line p0→p1 with the
 // clipping line specified by the clip point and normal.
-func isect(p0, p1, clip, norm Point) Point {
+func isect(p0, p1, clip, norm vg.Point) vg.Point {
 	// t = (norm · (p0 - clip)) / (norm · (p0 - p1))
-	t := p0.minus(clip).dot(norm) / p0.minus(p1).dot(norm)
+	t := p0.Sub(clip).Dot(norm) / p0.Sub(p1).Dot(norm)
```

Comment:
That makes their use a lot more verbose. These names are fine. They are mostly for internal use anyway.